live-editing/SharedAssetsGenerator.ts

```diff
@@ -19,9 +19,11 @@ const SAMPLES_PACKAGE_DEPENDENCIES = [
     "core-js",
     "hammerjs",
     "igniteui-angular",
+    "igniteui-angular-charts",
```

Comment:
@mharrington622 I thought you were going to figure out how to use a different filter for the charts stuff?

And I don't think we want the non-charts samples to have to load the charts stuff.

Reply:
Yeah, I just haven't gotten to that yet. Pam was asking about the stackblitz button, so I figured I would at least get this much committed first.  I'm still working on getting only the dependencies that are needed.

Reply:
I'm not positive that we can leave this in an intermediate state since it will create a bad experience if it ends up being left that way. I think its best to fix it fully first.

Reply:
@gmurray81 & @mharrington622, you could log this as a separate issue int this repository and assign it to me. I can take care of it. I can introduce a new property to the sample configuration specifying which packages to have as dependencies.
Side question - is this package (igniteui-angular-charts) going to stay public? I thought it will be private.